packages/core/src/baseclient.ts

```diff
@@ -139,6 +140,24 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
   public captureEvent(event: Event, hint?: EventHint, scope?: Scope): string | undefined {
     let eventId: string | undefined = hint && hint.event_id;
 
+    if (this._options.autoSessionTracking) {
+      const eventType = event.type || 'event';
+      const isException = eventType === 'event';
+
+      // If the event is of type Exception, then a request session should be captured
+      if (isException && scope) {
```

Comment:
For the record, here we should be checking whether the event has any exceptions.
For example, `captureEvent` with just a `message` should not report an errored session.

And we also need to handle explicit calls to `captureException`.

Reply:
Yep I am aware 
this is the non updated PR yet, I will move it to draft .. sorry for the confusion